Fundamentals/Setup/Server-Setup/azure-web-apps.md

```diff
@@ -77,3 +77,7 @@ The quickest way to get to your logs is using the following URL template and rep
 `https://{app}.scm.azurewebsites.net/api/logstream`
 
 You can also find this in the KUDU console by clicking **Advanced Tools** > **Log Stream** on the Web App in the Azure Portal.
+
+### Web App secret management
```

Comment:
How about changing the heading to H2 instead?
Doesn't sound like something that's related to "Web worker migrations" - but I might be wrong? 😄 

```suggestion
## Web App secret management
```